macros/dbt_utils/datetime/date_spine.sql

```diff
@@ -0,0 +1,119 @@
+{% macro sqlserver__date_spine_sql(start_date, end_date, datepart) %}
```

Comment:
this is the failure from Azure SQL. looking at `dbt-utils`'s [date_spine](https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/datetime/date_spine.sql) and [the model that's used to test `date_spine`](https://github.com/fishtown-analytics/dbt-utils/blob/master/integration_tests/models/datetime/test_date_spine.sql), it looks like we need to re-arrange the params.
>  ('42000', '[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Invalid parameter 1 specified for datediff. (1023) (SQLExecDirectW)')

```suggestion
{% macro sqlserver__date_spine_sql(datepart, start_date, end_date) %}
```